app/views/statuses/_group.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="status-group">
+  <%= render 'statuses/status', status: group %>
+
+  <div class="status-items">
```

Comment:
It seems that the CSS is using BEM. Following the BEM rules should not this be `status-group__items` since this class only have effect in the presence of the top level element?